odoo/addons/test_new_api/tests/test_new_fields.py

```diff
@@ -2361,3 +2361,258 @@ def test_delete_m2o_reference_records(self):
         # fake record to emulate the unlink of a non-existant record
         foo = m.browse(1 if not ids[0] else (ids[0] + 1))
         self.assertTrue(foo.unlink())
+
+
+@common.tagged('selection_ondelete_base')
+class TestSelectionOndelete(common.TransactionCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.MODEL_BASE = 'test_new_api.model_selection_base'
+        cls.MODEL_REQUIRED = 'test_new_api.model_selection_base_required'
```

Comment:
`setUpClass` is overkill for setting two constants on the class.

Reply:
oops, I changed this when you told me but reverted my code when we were fixing the tests